matlab_functions/plotScripts/Plot_BasicSimulationTimePlot.m

```diff
@@ -27,11 +27,11 @@
     ax(end+1)=subplot_tight(5,1,2,[plotmargins.vert plotmargins.horiz]);
     plot(time, flightdata.irr_array(3,:) * 180 / pi());
     ylabel('Angles');
-    legend('Sun incidence angle [�]');
+    legend('Sun incidence angle [°]');
     
     % Battery state
     subplot_tight(5,1,3,[plotmargins.vert plotmargins.horiz]);
-    E_bat_max = plane.bat.m * params.bat.e_density;
+    E_bat_max = flightdata.m_bat * params.bat.e_density;
```

Comment:
Oh yes, maybe it was overwritten by my custom code.

I just wanted to integrate flexibility into this function,
So that user can select any part of flightdata array, and use this function.

I am curious why this function accepts 'environment' variable as input, even though it is not used in this function? Is this some kind of practice in coding for huge projects?

So, I am willing to follow your instruction, since I might not fully understand your intention for every code you wrote.

Reply:
OK.

Note: The use of the "environment" variable is only a leftover from a previous implementation. I will remove this in a next PR.